examples/Script/script_json_communication.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import depthai as dai
+import json
+
+pipeline = dai.Pipeline()
+
+xin = pipeline.create(dai.node.XLinkIn)
+xin.setStreamName('in')
+
+script = pipeline.create(dai.node.Script)
+xin.out.link(script.inputs['in'])
+script.setScript("""
+import json
+
+# Recieve bytes from the host
```

Comment:
There's a typo @Erol444

Reply:
And @Erol444 indent the Script code if you'll be fixing this :)